src/components/tx-flow/common/TxNonce/index.tsx

```diff
@@ -102,6 +103,14 @@ const TxNonceForm = ({ nonce, recommendedNonce }: { nonce: number; recommendedNo
         },
       }}
       render={({ field, fieldState }) => {
+        if (readOnly) {
+          return <>{nonce}</>
+        }
+
+        console.log(field, field.value)
```

Comment:
```suggestion

```

Reply:
whoops. Return of the classic console log leftover 🤦